### PR TITLE
Version updates for adding new APIs

### LINKED ISSUE #31893: Include SqlClient update in 2.2 packaging
#31039 was incomplete. 
Per https://github.com/dotnet/corefx/pull/31039#issuecomment-415154728 please increment version and include in package list.
I think we can consider this already approved by shiproom.

@AfsanehR

### PR DESCRIPTION
**Pull request by AfsanehR**
Fixes #31893 
Addition of AccessToken APIs required increasing the versions.

### CODE REVIEW

File: src/System.Data.SqlClient/dir.props [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.4.0.0</AssemblyVersion>
```
Comment (weshaggard) [inline src/System.Data.SqlClient/dir.props]: You also need to set the PackageVersion here, similar to https://github.com/dotnet/corefx/blob/release/2.0.0/src/System.Data.SqlClient/dir.props#L5. Set it 4.6.0 and update the version in master to 4.7.0.

**Comment by weshaggard** [comment]
You also need to add the System.Data.SqlClient package to the list of packages to build in this branch, similar to https://github.com/dotnet/corefx/commit/93055b13249da58e01ea19f91211b8659d3ad178#diff-cb73a137838f73872d476fa5146bfedeR27.

File: src/System.Data.SqlClient/dir.props (hunk 1 above)
Comment (AfsanehR) [reply src/System.Data.SqlClient/dir.props]: @weshaggard Thanks. Updated it. Will update the master after this pr is merged in another pr.

File: pkg/Microsoft.Private.PackageBaseline/packageIndex.json [hunk 2]
```diff
@@ -1267,7 +1267,7 @@
         "4.1.0.0": "4.1.0",
         "4.1.1.0": "4.3.0",
         "4.2.0.0": "4.4.0",
-        "4.4.0.0": "4.5.0"
+        "4.5.0.0": "4.5.0"
```
Comment (weshaggard) [inline pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: This is now wrong. Did you update it manually? Try running an update from the pkg project build `msbuild /t:UpdatePackageIndex`

Comment (weshaggard) [reply pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: No run it from inside the src\System.Data.SqlClient\pkg directory. You will need to do at least one full all configuration build from the root `build.cmd -allconfigurations` in order to enable individual package building.

Comment (AfsanehR) [reply pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: I did run with ```build.cmd -allConfigurations ``` command in the root directory and then ran the ```msbuild /t:UpdatePackageIndex``` command on SqlClient package directory, still seeing the failure.  The changes in the packageIndex.json file is also not related to versions.

**Comment by AfsanehR** [comment]
@dotnet-bot Test Packaging All Configurations x64 Debug Build

**Comment by AfsanehR** [comment]
@dotnet-bot Test Linux-musl x64 Debug Build

File: src/packages.builds [hunk 3]
```diff
@@ -26,6 +26,9 @@
     <Project Include="$(MSBuildThisFileDirectory)..\pkg\Microsoft.NETCore.Platforms\Microsoft.NETCore.Platforms.builds">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
+    <Project Include="*\pkg\**\System.Data.SqlClient.pkgproj" Condition="'$(SkipManagedPackageBuild)' != 'true' AND '$(BuildAllConfigurations)' == 'true'">
```
Comment (weshaggard) [inline src/packages.builds]: For this just set it to:
```
<Project Include="$(MSBuildThisFileDirectory)\System.Data.SqlClient\pkg\System.Data.SqlClient.pkgproj">
      <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
    </Project>
```

File: pkg/Microsoft.Private.PackageBaseline/packageIndex.json (hunk 2 above)
Comment (weshaggard) [reply pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: OK just manually update this section to be:

```
        "4.4.0.0": "4.5.0",
        "4.5.0.0": "4.6.0"
```

File: src/System.Data.SqlClient/dir.props (hunk 1 above)
Comment (weshaggard) [reply src/System.Data.SqlClient/dir.props]: @AfsanehR please go ahead and make a PR for master to update these versions.

**Pull request merged**